dev/archery/archery/cli.py

```diff
@@ -772,6 +772,64 @@ def _set_default(opt, default):
               help=("Substring for test names to include in run, "
                     "e.g. -k primitive"))
 def integration(with_all=False, random_seed=12345, **args):
+    """If you don't specify the "--target-implementations" option nor
```

Comment:
We can use `click.command(help)` or docstring to show command explanation for `archery integration --help`.
I use docstring for it.